nimble/host/include/host/ble_gap.h

```diff
@@ -1582,6 +1582,14 @@ struct ble_gap_ext_adv_params {
 
     /** Advertising Set ID */
     uint8_t sid;
+
+#if MYNEWT_VAL(BLE_EXT_ADV_V2)
```

Comment:
since below fields are OK to be `0` which means, `host has not preference` I would skip `BLE_EXT_ADV_V2` .

Instead, host could choose to use V2 or V1 based on the Read Supported Commands result.

@sjanc @andrzej-kaczmarek  what do you think?

Reply:
Hi @rymanluk  
 
> Instead, host could choose to use V2 or V1 based on the Read Supported Commands result. 

Just to understand this comment, are you implying that use V2 directly if its support is present ? ( i.e. don't use V1  in such cases ? )  Will host not able to use V1, if controller has support for V2 ?

Reply:
Hi @rymanluk , i have updated the PR to not use the BLE_EXT_ADV_V2 macro and rather decide based on Read Supported Commands.